go.mod

```diff
@@ -7,6 +7,7 @@ replace github.com/VictoriaMetrics/fastcache => ./fastcache
 replace github.com/ethereum/go-ethereum => ./go-ethereum
 
 require (
+	github.com/EspressoSystems/espresso-sequencer-go v0.0.2-0.20231213053144-08be545d123d
```

Comment:
@ImJeremyHe Is it possible to add the new dependency without updating the old ones? The reason I ask is that it will cause merge conflict with upstream updating the dependencies.

Reply:
I will check that.

Reply:
Thanks for checking. I think we just keep it then and fix the conflicts when we have to.